docs/context/ref/Email.md

```diff
@@ -0,0 +1,60 @@
+---
+id: Email
+sidebar_label: Email
+title: Email
+hide_title: true
+---
+# `Email`
+
+A collection of information to be used to initiate an email with a Contact or ContactList
+
+## Type
+
+`fdc3.email`
+
+## Schema
+
+https://fdc3.finos.org/schemas/next/email.schema.json
+
+## Details
+
+| Property          | Type                                  | Required | Example Value       |
+|-------------------|---------------------------------------|----------|---------------------|
+| `type`            | string                                | Yes      | `'fdc3.email'` |
+| `recipients`      | fdc3.contact or fdc3.contactList      | Yes      | { type: "fdc3.contact", name: "John Doe", id: { "email": "john@sample.com"}} |
```

Comment:
Given that this could be singular or plural I wonder if it should be `to` instead of `recipients`

Reply:
I guess I talk about the "recipients of an email" regardless of whether there is one or more; it's just a generic phrase. The issue with to: is it is a technical term in SMTP which then invites "what about cc: and bcc:" (also technical terms).

Reply:
fair enough - we did discuss this (I now remember)